examples/nomad-consul-ami/nomad-consul.json

```diff
@@ -68,10 +58,19 @@
     ],
     "only": ["amazon-linux-ami"]
   },{
-    "type": "shell",
-    "inline": [
-      "git clone --branch {{user `consul_module_version`}} https://github.com/hashicorp/terraform-aws-consul.git /tmp/terraform-aws-consul",
-      "/tmp/terraform-aws-consul/modules/install-consul/install-consul --version {{user `consul_version`}}"
-    ]
+    "type": "file",
+    "source": "{{template_dir}}/../../../terraform-aws-nomad",
+    "destination": "/tmp",
+    "pause_before": "30s"
+  },{
+    "type": "file",
+     "source": "setup_nomad_consul.sh",
+     "destination": "/tmp/"
+  },{
+     "type": "shell",
+     "environment_vars": [ "NOMAD_VERSION={{user `nomad_version`}}",
+                           "CONSUL_VERSION={{user `consul_version`}}",
+                           "CONSUL_MODULE_VERSION={{user `consul_module_version`}}"],
```

Comment:
JSON formatting puts the square brackets on their own lines.

Reply:
[Done: Improved formatting for the whole file](https://github.com/hashicorp/terraform-aws-nomad/pull/11/commits/1bbc5b0a2bfb42c99d3992ab7c258b4911768769)